src/vivarium_gates_mncnh/data/lbwsg_paf.yaml

```diff
@@ -27,12 +28,12 @@ configuration:
         end:
             year: 2022
             month: 1
-            day: 2
-        step_size: 3 # Days
+            day: 10
+        step_size: 8 # Days
     population:
-        population_size: 389_992
+        population_size: 194_996

```

Comment:
We're halving the population because we're putting twice as many simulants in each sex-specific LBWSG category as before. Since we want to keep the number of simulants in each category the same, we're changing the population.